dips/dip-158.md

```diff
@@ -53,71 +47,55 @@ See [Appendix A](#appendix-a---prerequisite-sharing-common-payment-identifier-an
+7. The wallet initializes the payment protocol (off-chain) which will end (if successful) with the funds being charged. Successful completion implies that the receiving VASP was satisfied with the payer information provided and the sender VASP was satisfied with the merchant information provided. **Note** that if the payment amount exceeds the amount that requires a dual attestation signature to be submitted to the blockchain, it must be part of the on-chain transaction submitted  
 
 ### **Charge Payment Type** 
 Charge payment stipulates an immediate capture of funds. This means that a successful off-chain negotiation for such payment will result in funds being captured.
-
 As soon as the funds are captured, i.e. the sender agreed to the payment terms, the sender VASP is expected to submit a corresponding transaction on-chain and actually transfer the funds. 
 
-
-### **Auth / Capture Payment Type**
-Auth / Capture payment, as the name suggests, means that the payment is divided into two steps:
-* Authorization
-* Capture
- 
-The Authorization step ends when the funds are locked by the sender VASP so they can later be captured by the receiver VASP. Note that the capture may occur several days after a successful authorization. It is up to the receiver VASP to decide on the timing. In addition, the receiver may choose to capture only a partial amount rather than the full amount which was authorized. 
-
-This payment type is useful for delayed fulfillment or pre-authorizing an expected amount to ensure that an amount can be captured after services are rendered. 
-
-Off-chain negotiations for Auth / Capture payment are divided into two steps accordingly. First, off-chain negotiations will result in funds being authorized, i.e. locked by the wallet. Later, a separate off-chain negotiation triggered by the receiving VASP will capture the funds. Note that for the time being, multi-capture is not supported and there can be only a single capture request.
-
-Successful authorization will not necessarily be followed by a successful capture. It is possible that the receiver will choose to cancel the authorization and essentially void the payment before it was captured. One possible reason to void a payment is that the products for which an amount was locked are out of stock and cannot be fulfilled.
+### **Other Payment Types (future)**
+Protocols supporting other eCommerce payments flows will require additional payment types which will be be defined in future DIPs.
+Some examples for such flows are:
+* Auth / Capture
+* Subscription payment
+* Recurring payments
```

Comment:
I think we discussed this elsewhere but are these fundamentally different?
Should we change this to Other P2M Flows and include user-driven refunds (disputes) and merchant-driven refunds?

Reply:
Refunds, unlike other payment types, needs to be specified/delivered as soon as possible, as part of phase 1.
I hope that when the first developer reads DIP-158, the refunds DIP will already be readily available. 
Other payment types (e.g. subscription, recurring, auth/capture) appear in DIP-158 only to state these are currently not supported and will be specified in future DIPs.

Reply:
@davidiw - regarding the conflicts, not sure why the two changes I made appear as conflicts.
Both changes (specified below) should be merged.
1. Updating the link to the sequence diagram to be that of a QR scan rather than the web redirect
2. Adding a comment about the error list being non-exhaustive

Reply:
@YinonFirstDAG , for conflicts, the best solution is to rebase and push an update. I've noticed this repository to be a bit flaky.

Reply:
I will try a new PR from a fresh branch.
Closing this one.